test/e2e/common/setup-kubestellar.sh

```diff
@@ -140,6 +140,8 @@ if [ "$use_release" != true ]; then
   OCM_TRANSPORT_PLUGIN_DIR="$(pwd)"
   pwd
   echo "replace github.com/kubestellar/kubestellar => ${KUBESTELLAR_DIR}/" >> go.mod
+  echo "replace go.opentelemetry.io/otel/exporters/otlp/otlptrace/otlptracegrpc => go.opentelemetry.io/otel/exporters/otlp/otlptrace/otlptracegrpc v1.14.0" >> go.mod # otel broke semver rules
```

Comment:
Should we really do it from the shell script ?  Why not just add it directly into the go.mod and set the version ?

Reply:
That `go.mod` file is not in this git repository; it is in the ks/OTP repository.

Reply:
but it's a fixed change, why not changing it in that repo ?

Reply:
We can do that when we make the next ks/OTP release. Until then, we need the new replacement statements to be injected from the ks/ks repo.